src/components/routes/qdm/SDEPage.tsx

```diff
@@ -1,6 +1,205 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  RadioButton,
+  MadieDiscardDialog,
+  Toast,
+} from "@madie/madie-design-system/dist/react";
+import { useFormik, useFormikContext } from "formik";
+import { Measure } from "@madie/madie-models";
+import { measureStore, routeHandlerStore } from "@madie/madie-util";
+import useMeasureServiceApi from "../../../api/useMeasureServiceApi";
+import "./SDEPage.scss";
+
+interface SDEForm {
+  sdeIncluded: string;
+}
 
 const SDEPage = () => {
-  return <div data-testId="sde-page" />;
+  const [measure, setMeasure] = useState<any>(measureStore.state);
+  const measureServiceApi = useMeasureServiceApi();
+  const { updateMeasure } = measureStore;
+  const { updateRouteHandlerState } = routeHandlerStore;
+  const [discardDialogOpen, setDiscardDialogOpen] = useState(false);
+  const [toastOpen, setToastOpen] = useState<boolean>(false);
+  const [toastMessage, setToastMessage] = useState<string>("");
+  const [toastType, setToastType] = useState<string>("danger");
+  const [currentSdeIncluded, setCurrentSdeIncluded] =
+    useState<boolean>(undefined);
+
+  useEffect(() => {
+    const subscription = measureStore.subscribe(setMeasure);
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+
+  useEffect(() => {
+    if (measure && measure.testConfiguration) {
+      setCurrentSdeIncluded(measure.testCaseConfiguration.sdeIncluded);
+    }
+  }, [measure]);
+
+  const formik = useFormik({
+    initialValues: {
+      sdeIncluded:
+        String(measure?.testCaseConfiguration?.sdeIncluded) || "true",
```

Comment:
not sure if this condition works here, if measure.testCaseConfiguration.sdeIncluded is undefined, wouldn't formik.values.sdeIncluded be undefined ?

Also if undefined or null, shouldn't the default value be false ?

Reply:
I changed the default to false